### DISCUSSION TITLE
Addition of GraalVM dependencies in IntelliJ

### DISCUSSION THREAD

**Discussion by andrii0lomakin** [beehive-lab/TornadoVM#508] [Q&A]
Good day.
Documentation on how to add GraalVM dependencies in the TornadoVM maven project is not clear. It can be done by Gradle DSL easily, but I did not find any guidelines in the project documentation on how typically that is done by TornadoVM developers in the case of a maven project.
I can figure it out myself, but it would save time if  you help with two questions :

1. How do you add GraalVM jars as dependencies in a robust way (will not be overridden every time the maven project is reexported)?
2. How to handle quirks in the JVM module system if there will be any.

**Comment by andrii0lomakin** [comment]
I created an issue for my colleagues, I hope it will be resolved soon. https://youtrack.jetbrains.com/issue/IDEA-357231/IDEA-does-not-take-into-account-module-info.java .

**Reply by jjfumero** [reply]
That would be great, thanks.
